crates/next-dev-tests/Cargo.toml

```diff
@@ -0,0 +1,57 @@
+[package]
+name = "next-dev-tests"
```

Comment:
Can we remove any deps from `next-dev/Cargo.toml`?

Reply:
Some of them are used by benchmarks, so I removed only unused ones